docs/package_info/glossary.rst

```diff
@@ -0,0 +1,46 @@
+Glossary
+========
+
+.. _access_token:
+
+* ``Access Token``: A token that is obtained from authenticating with Reddit.
+  The token lasts for an hour, so multiple tokens may need to be obtained.
+
+.. _camel_case:
+
+* ``Camel Case``: A type of variable notation where multiple words are
+  seperated by upper case. For example, the glossary entry for
+  :ref:`Access Token <access_token>` would be written as ``accessToken`` in
+  Camel Case.
+
+.. _creddit:
+
+* ``Creddit``: Back when the only award was ``Reddit Gold``, a creddit was
+  equal to one month of Reddit Gold. Creddits have been converted to
+  ``Reddit Coins``. See :ref:`this <gild>` for more info about the old Reddit
+  Gold system.
+
+.. _fullname:
+
+* ``Fullname``: The fullname of an object is the object's type followed by an
+  underscore and it's base-36 id.
```

Comment:
This would be great with an example or two. Like, explain that `t3_ezz3td` means the submission with ID "ezz3td."

Also, that should be "its" (no apostrophe).